spec/spec_helper.rb

```diff
@@ -8,4 +8,10 @@
 
 # Requires supporting ruby files with custom matchers and macros, etc,
 # in spec/support/ and its subdirectories.
-Dir[File.join(File.dirname(__FILE__), 'support/**/*.rb')].sort.each { |f| require f }
\ No newline at end of file
+Dir[File.join(File.dirname(__FILE__), 'support/**/*.rb')].sort.each { |f| require f }
+
+require 'spree/api/testing_support/setup'
+
+RSpec.configure do |config|
+  config.extend Spree::Api::TestingSupport::Setup, type: :request
```

Comment:
@kgorazd could we also move this into spree dev tools https://github.com/spree/spree_dev_tools/blob/master/lib/spree_dev_tools/rspec/support/spree.rb ?